parsers.dockerfile

```
@@ -0,0 +1,12 @@
+# Container image that executes the parser tests 
+# Build with "$ docker build -f parsers.dockerfile -t parsers . "
+# Run with  "$ docker run parsers US-CAL-CISO production"
+# If needed, pass any token env var using "docker run -e TOKEN=<token>"
+
+FROM python:3.10
```

```suggestion
FROM python:3.8
```
This should be python 3.8